build/RELEASING.md

```diff
@@ -2,63 +2,19 @@
 
 Only for Maintainers.
 
- 1. Decide the tag name (version name) to be released.
-    eg: 1.0.0-rc2, 1.0.0 etc.
+## Prerequisites
 
- 2. Run the following PowerShell from the root of the
-    repo to get combined changelog (to be used later).
+- Install [GitHub CLI](https://cli.github.com/)
 
-    ```powershell
-        $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
-        foreach ($changelog in $changelogs)
-        {
-         Add-Content -Path .\combinedchangelog.md -Value "**$($changelog.Directory.Name)**"
-         $lines = Get-Content -Path $changelog.FullName
-         $started = $false
-         $ended = $false
-         foreach ($line in $lines)
-             {
-                if($line -like "## Unreleased" -and $started -ne $true)
-                {
-                  $started = $true
-                }
-                elseif($line -like "## *" -and $started -eq $true)
-                {
-                  $ended = $true
-                  break
-                }
-                else
-                {
-                    if ($started -eq $true)
-                    {
-                        Add-Content -Path .\combinedchangelog.md $line
-                    }
-                }
-             }
-        }
-    ```
-
-    This generates combined changelog to be used in Github release.
-    Once contents of combined changelog is saved somewhere,
-    delete the file.
-
- 3. Run the following PowerShell script from the root of the repo.
-    This updates all the changelog to have release date for the
-    current version being released.
-    Replace the version with actual version.
-    The actual version would be the tag name from step1.
+## Steps
 
-    ```powershell
-         $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
-        foreach ($changelog in $changelogs)
-        {
-         (Get-Content -Path $changelog.FullName) -replace "Unreleased", "Unreleased
+1. Decide the tag name (version name) to be released.
+   eg: 1.0.0-rc2, 1.0.0 etc.
 
-    ## 1.0.0-rc2
+1. Obtain the API key from [nuget.org](#TODO) (Only maintainers have access)
 
-    Released $(Get-Date -UFormat '%Y-%b-%d')" | Set-Content -Path $changelog.FullName
-        }
-    ```
+1. Run the following script from the root of the repository
+   `./build/Release.ps1 <TagName> <NuGetApiKey> <CoreComponents> <NonCoreComponents>`
```

Comment:
I'm not familiar with how to set a boolean value in PS command. What would a command be like if you want to release, say, the core component?

Reply:
`$True` and `$False`